starter/src/main/java/com/starter/fullstack/rest/InventoryController.java

```diff
@@ -29,9 +32,17 @@ public InventoryController(InventoryDAO inventoryDAO) {
    * Find Products.
    * @return List of Product.
    */
-  @GetMapping
+  @GetMapping("/inventory")
   public List<Inventory> findInventories() {
     return this.inventoryDAO.findAll();
   }
+
+  /**
+   *
+   */
+  @PostMapping("/inventory")
```

Comment:
Take a look at this documentation to see if you need to specify the /inventory route on your get and post mapping annotations even though the RequestMapping for the class is already set to /inventory

https://docs.spring.io/spring-framework/reference/web/webflux/controller/ann-requestmapping.html